pkg/kubectl/cmd/get.go

```diff
@@ -150,6 +150,9 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 	allNamespaces := cmdutil.GetFlagBool(cmd, "all-namespaces")
 	showKind := cmdutil.GetFlagBool(cmd, "show-kind")
 	mapper, typer := f.Object()
```

Comment:
it's possible the test typer we mocked up in NewMixedFactory is wrong, or it's possible something else is wrong, but we need to figure out the problem there and fix it, not swap in a fake typer here for test